app/routes/deploy.py

```diff
@@ -55,10 +55,26 @@ async def deploy():
         entry["image"] = "bitswan/pipeline-runtime-environment:latest"
 
         network_mode = None
+        secret_groups = []
         if pipeline_conf:
             network_mode = pipeline_conf.get(
                 "docker.compose", "network_mode", fallback=conf.get("network_mode")
             )
+            secret_groups = pipeline_conf.get("secrets", "groups", fallback="").split(
+                " "
+            )
+        for secret_group in secret_groups:
+            gitops_dir = os.environ.get(
+                "BITSWAN_GITOPS_DIR",
+                os.path.join(os.environ.get("HOME"), ".config/bitswan/local-gitops/"),
+            )
+            if os.path.exists(gitops_dir):
```

Comment:
Because gitops is running in container the `gitops_dir` have to be set as gitops path on host. This won't work then because the path won't exist in the container